charts/mlp/.helmignore

```diff
@@ -0,0 +1,25 @@
+# Patterns to ignore when building packages.
```

Comment:
Is it possible to make it DRY? Can we have a single `.helmignore` in the root of the `./charts` directory and use symlinks (if necessary) from the individual charts to point to this "master-copy" of the `.helmignore` file?

Reply:
i'll check if it's possible.

Reply:
there is no helm-supported way to specify the `.helmignore` file location. Added it as a symlink. Packaging / dependency related commands are taking the contents of the linked file into consideration

```
walk.go:74: found symbolic link in path: helm-charts/charts/mlp/.helmignore resolves to helm-charts/.helmignore. Contents of linked file included and used
```